README.md

```diff
@@ -14,54 +22,60 @@ For documentation on previous versions of the Molecular Targets Platform, please
 
 | Dataset | Number of Samples |
 | --- | --- |
-| PBTA | 2,984 |
-| TARGET | 4,489 |
-| GMKF | 884 |
+| PBTA | 4,164 |
+| TARGET | 5,252 |
+| GMKF | 881 |
+| DGD | 1,799 |
 | GTEx | 17,382 |
-| TCGA | |
+| TCGA | 10,414 |
 
-For expanded descriptions of the datasets, please see the About page on the Molecular Targets Platform LINK TO ABOUT DATA SECTION ON SITE.
+For expanded descriptions of the datasets, please see the About page on the Molecular Targets Platform **LINK TO ABOUT DATA SECTION ON SITE**.
 
 ---
 
-## Somatic_Alterations
+## DNA_Sequencing
 
 ### Data Processing
 
 #### DNA-seq Alignment and Haplotype Calling Workflow
 
-For both whole genome, whole exome, and targeted panel DNA sequencing, the workflow begins by flagging duplicates and aligning fastq files, or re-aligning previously aligned BAMs, to the reference genome GRCh38 using bwa mem. The majority of Pediatric Molecular Target data is paired-end, but single end methods are provided if you want to apply the pipeline to your own data. Sequencing quality is checked using FastQC and tumor/normal pairs are double-checked to confirm they are from the same individual using NGSCheckMate. Variants are called using GATK4 HaplotypeCaller. For more details on the alignment or to run the CAVATICA app yourself, see the GitHub release at [Kid's First Alignment and Haplotype Calling Workflow](https://github.com/kids-first/kf-alignment-workflow) and the [CAVATICA App](https://cavatica.sbgenomics.com/public/apps/cavatica/apps-publisher/kfdrc-alignment-workflow). Once in the Cavatica workflow page, please click on the "Read All" link to open up the full documentation.
+For both whole genome, whole exome, and targeted panel DNA sequencing, the workflow begins by flagging duplicates and aligning fastq files, or re-aligning previously aligned BAMs, to the reference genome GRCh38 using bwa mem. The majority of Pediatric Molecular Target data is paired-end, but single end methods are provided if you want to apply the pipeline to your own data. Sequencing quality is checked using FastQC and tumor/normal pairs are double-checked to confirm they are from the same individual using NGSCheckMate. For more details on sample identity confirmation please see the [Kid's First NGS Checkmate Workflow](https://github.com/kids-first/ngs_checkmate_wf). Variants are called using GATK4 HaplotypeCaller. For more details on the alignment or to run the CAVATICA app yourself, see the GitHub release at [Kid's First Alignment and Haplotype Calling Workflow](https://github.com/kids-first/kf-alignment-workflow) and the [CAVATICA App](https://cavatica.sbgenomics.com/public/apps/cavatica/apps-publisher/kfdrc-alignment-workflow). Once in the Cavatica workflow page, please click on the "Read All" link to open up the full documentation.
```

Comment:
```suggestion
For whole genome, whole exome, and targeted panel DNA sequencing, the workflow begins by flagging duplicates and aligning fastq files, or re-aligning previously aligned BAMs, to the reference genome GRCh38 using bwa mem. The majority of Pediatric Molecular Target data is paired-end, but single end methods are provided if you want to apply the pipeline to your own data. Sequencing quality is checked using FastQC and tumor/normal pairs are double-checked to confirm they are from the same individual using NGSCheckMate. For more details on sample identity confirmation please see the [Kid's First NGS Checkmate Workflow](https://github.com/kids-first/ngs_checkmate_wf). Variants are called using GATK4 HaplotypeCaller. For more details on the alignment or to run the CAVATICA app yourself, see the GitHub release at [Kid's First Alignment and Haplotype Calling Workflow](https://github.com/kids-first/kf-alignment-workflow) and the [CAVATICA App](https://cavatica.sbgenomics.com/public/apps/cavatica/apps-publisher/kfdrc-alignment-workflow). Once in the Cavatica workflow page, please click on the "Read All" link to open up the full documentation.
```

Can you update this PR to be one sentence per line for easier review?

Reply:
Sorry, I didn't expect such large blocks of text, but anything that is replacing a large block of text with another large block of text was pulled from main, was already approved, and doesn't need review. Despite that, I did try putting sentences as below and I'm not sure that really helps. If it would make it easier and I can do that and then submit a second PR putting the paragraphs back together.
![Screen Shot 2022-07-25 at 4 58 37 PM](https://user-images.githubusercontent.com/26015546/180873096-6735e726-60b5-4805-99ab-6516d50e6086.png)

Reply:
Thanks, it would make it easier for any future reviews, so if you can do that here, that'll help later!

Reply:
I broke out all the paragraphs into single sentences per line so hopefully review will be easier in the future!